subsys/bluetooth/controller/ll_sw/ull_sched.c

```diff
@@ -560,6 +594,39 @@ static void win_offset_calc(struct ll_conn *conn_curr, uint8_t is_select,
 }
 #endif /* CONFIG_BT_LL_SW_LLCP_LEGACY */
 #endif /* CONFIG_BT_CTLR_CONN_PARAM_REQ */
+
+static void after_cen_offset_get(uint16_t conn_interval, uint32_t ticks_slot,
+				 uint32_t ticks_anchor,
+				 uint32_t *win_offset_us)
+{
+	uint32_t ticks_anchor_offset = ticks_anchor;
+	int err;
+
+	err = ull_sched_after_cen_slot_get(TICKER_USER_ID_ULL_LOW, ticks_slot,
+					   &ticks_anchor_offset,
+					   win_offset_us);
+	if (err) {
+		return;
+	}
+
+	if ((ticks_anchor_offset - ticks_anchor) & BIT(HAL_TICKER_CNTR_MSBIT)) {
+		*win_offset_us -= HAL_TICKER_TICKS_TO_US(
+			ticker_ticks_diff_get(ticks_anchor,
+					      ticks_anchor_offset));
+	} else {
+		*win_offset_us += HAL_TICKER_TICKS_TO_US(
+			ticker_ticks_diff_get(ticks_anchor_offset,
+					      ticks_anchor));
+	}
+
+	if ((*win_offset_us & BIT(31)) == 0) {
```

Comment:
What is bit 31 here?

Reply:
Used to test Most Significant Bit of uint32_t. I could not find a suitable define.

Reply:
The comment helps :) So the MSBit indicates that the offset is in the past? 

Sounds like something we could define a (named) check for, but if it is just used here then the comment is good enough.